.kokoro/tests/run_tests.sh

```diff
@@ -104,9 +104,11 @@ for file in **/requirements.txt; do
     nox -s "$RUN_TESTS_SESSION"
     EXIT=$?
 
-    # If this is a periodic build, send the test log to the Build Cop Bot.
-    # See https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop.
-    if [[ $KOKORO_BUILD_ARTIFACTS_SUBDIR = *"periodic"* ]]; then
+    # If REPORT_TO_BUILD_COP_BOT is set to "True", send the test log
+    # to the Build Cop Bot.
+    # See:
+    # https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop.
+    if [[ "${REPORT_TO_BUILD_COP_BOT:-}" == "True" ]]; then
```

Comment:
nit: I _think_ the `:-` is redundant since there are already quotes around the string (but I'm by no means a bash expert.)
```suggestion
    if [[ "${REPORT_TO_BUILD_COP_BOT}" == "True" ]]; then
```

Reply:
Now we don't have the `-u` option, but when we add it, this script still works.

Not planning to add `-u` option, but it doesn't do any harms. WDYT?